src/index.js

```diff
@@ -34,60 +34,66 @@ function findTodaysDate() {
   today = `${yyyy}/${mm}/${dd}`;
 }
 
+function formatSelectedDate(date) {
```

Comment:
Oh this is interesting, I've never used regex. Does replace need to be run 2x? IT might just replace the first dash.

Reply:
I was going to double up on `replace()` but found regex while researching it.  The regex lets me replace both without having to chain `replace()` twice.  I was pretty excited about it so I'm glad you noticed it!